source/guides/assets/overview.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Lotus - Guides - Assets Overview
+---
+
+# Assets
+
+Lotus supports powerful features for web assets.
+
+## Configuration
+
+Each application can have its own separated assets settings in application configuration.
+
+### Compile Mode
+
+Toggle this value, to determine if the application must preprocess or copy assets from sources to public directory.
+It's turned on by default in _development_ and _test_ environments, but turned off for _production_.
+
+```ruby
+# apps/web/application.rb
+module Web
+  class Application < Lotus::Application
+    configure do
+      # ...
+      assets do
+        # compile true, enabled by default
+      end
+    end
+
+    configure :production do
+      compile false
+    end
+  end
+end
+```
+
+### Digest Mode
+
+In order to force browsers to cache the right copy of an asset, during the deploy, Lotus creates a copy of each file by [appending its checksum](/guides/command-line/assets) to the file name.
+
+We can control this feature via application configuration.
+It's turned on by default only in _production_ environment.
+
+```ruby
+# apps/web/application.rb
+module Web
+  class Application < Lotus::Application
+    configure do
+      # ...
+      assets do
+        # digest false, disabled by default
+      end
+    end
+
+    configure :production do
+      digest true
```

Comment:
@jodosha I think you have to envelop `digest true` with `assets` block, isn't it?